src/SprykerEco/Zed/Computop/Business/ComputopFacade.php

```diff
@@ -22,11 +22,6 @@ class ComputopFacade extends AbstractFacade implements ComputopFacadeInterface
 
     /**
      * {@inheritdoc}
-     *
-     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
-     * @param \Generated\Shared\Transfer\CheckoutResponseTransfer $checkoutResponseTransfer
-     *
-     * @return void
```

Comment:
If you take all away the tag is `@inheritdoc`. `{@inheritdoc}` is only used with the param and return tag still present.

Reply:
Done